adapters/core2p2p/block.go

```diff
@@ -17,6 +21,37 @@ func AdaptBlockID(header *core.Header) *spec.BlockID {
 	}
 }
 
+func AdaptSignature(sig []*felt.Felt) *spec.ConsensusSignature {
+	return &spec.ConsensusSignature{
+		R: AdaptFelt(sig[0]),
+		S: AdaptFelt(sig[1]),
+	}
+}
+
+func AdaptHeader(header *core.Header, commitments *core.BlockCommitments) *spec.BlockHeader {
+	return &spec.BlockHeader{
+		ParentHeader:     AdaptHash(header.ParentHash),
+		Number:           header.Number,
+		Time:             timestamppb.New(time.Unix(int64(header.Timestamp), 0)),
+		SequencerAddress: AdaptAddress(header.SequencerAddress),
+		ProofFact:        nil, // not defined yet
+		Receipts:         nil, // not defined yet
+		StateDiffs:       nil, // not defined yet
+		State: &spec.Patricia{
+			Height: uint32(header.Number),
```

Comment:
The `Height` here is the height of the patricia trie. So it should be a fixed `251`. I know, It is weird but, yeah.